src/styles/blocks/animation-name.scss

```diff
@@ -0,0 +1,17 @@
+@keyframes hour-rotate {
+  100% {
+    rotate: (360deg);
+  }
+}
+
+@keyframes minute-rotate {
+  100% {
+    rotate: (360deg);
+  }
+}
+
+@keyframes second-rotate {
+  100% {
+    rotate: (360deg);
+  }
+}
```

Comment:
you have 3 equal `@keyframes` mb make sense to use 1?

Reply:
rewrite your code and use one @keyframes